src/hooks.ts

```diff
@@ -16,24 +16,27 @@ export function useCallsDataSubscription(
   context: MulticallContext,
   chainId: number | undefined,
   calls: Array<Call | undefined>,
-  blocksPerFetch = 1
+  blocksPerFetch?: number
```

Comment:
Why not change this to `ListenerOptions`? Then you can pass that directly to the dispatcher, and deal with the defaults in the dispatch actions (in slice.ts).

Reply:
Sgtm done